### ISSUE TITLE
Generator and shortage do not provide new functionality

### ISSUE THREAD

**Issue by jnnr** [oemof/oemof-tabular#90]
... they are just aliases of Dispatchable. Should drop them with the next release.

**Comment by nailend** [comment]
Thats true but isn't tabular also a bit about having more specific 'names', then the generic one from solph?
I would fancy keeping them
